lib/cli/ui/stdout_router.rb

```diff
@@ -133,6 +146,26 @@ class << self
 
         NotEnabled = Class.new(StandardError)
 
+        def with_id(on_streams:)
+          unless on_streams.is_a?(Array) && on_streams.all? { |s| s.respond_to?(:write) }
+            raise ArgumentError, <<~EOF
+            on_streams must be an array of objects that respond to `write`
+            These do not respond to write
+            #{on_streams.reject { |s| s.respond_to?(:write) }.map.with_index { |s| s.class.to_s }.join("\n")}
+            EOF
+          end
+
+          require 'securerandom'
+          id = format("%05d", rand(100))
```

Comment:
This is a number from 0 to 99 padded out with "0" to 5 places. `format("%05d", rand(10**5))`

Reply:
fixed in 1.1.3